packages/analytics-browser/src/browser-client.ts

```diff
@@ -169,12 +170,11 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
     return this.config?.sessionId;
   }
 
-  setSessionId(sessionId: number) {
+  setSessionId(sessionId: number, shouldTrackNewCampaign?: boolean) {
```

Comment:
I'm happy we don't need to make this async, but think it could still make sense to return a Promise object to allow users to await session event tracking if desired. e.g.
`setSessionId(sessionId: number): { promise: Promise }`

Reply:
Done.